common/pom.xml

```diff
@@ -27,9 +27,9 @@
 
 	<artifactId>fhir-common</artifactId>
 	
-	<properties>
-	  <c3p0.version>0.9.5.4</c3p0.version>
-	</properties>
+		<properties>
+		<c3p0.version>0.9.5.4</c3p0.version>
```

Comment:
The indentation is still wrong here; we should not be really talking about indents this much in a review.